### PR TITLE
DPM-926

### PR DESCRIPTION
**Pull request by alrxy**

### CODE REVIEW

File: examples/stub/tests/stub_tests.cpp [hunk 1]
```diff
@@ -492,6 +492,113 @@ BOOST_FIXTURE_TEST_CASE(new_session_affl_test, stub_tester) try {
...
+//     auto casino_balance_before = get_balance(casino_name);
+//     auto player_balance_before = get_balance(player_name);
+
+//     auto player_bet = ASSET("5.00000 KEK");
+//     auto ses_id = new_game_session(game_name, player_name, casino_id, player_bet);
+
+//     BOOST_REQUIRE_EQUAL(get_balance(game_name), player_bet);
+
+//     game_action(game_name, ses_id, 0, {0});
+
```
Comment (justefg) [inline examples/stub/tests/stub_tests.cpp]: ?

File: sdk/include/game-contract-sdk/game_base.hpp [hunk 2]
```diff
@@ -348,14 +357,19 @@ class game : public eosio::contract {
             return;
         }
 
-        eosio::check(quantity.symbol == core_symbol, "invalid token symbol");
-
+        // token verifying
+        const auto platform = get_platform();
```
Comment (justefg) [inline sdk/include/game-contract-sdk/game_base.hpp]: it's not a mistake but I think we could get rid of `platform` variable

Comment (justefg) [reply sdk/include/game-contract-sdk/game_base.hpp]: also why is `get_token_contract` check not enough?

**Comment by justefg** [comment]
excellent job 👍

**Comment by alrxy** [comment]
> excellent job 👍

thanks

File: sdk/include/game-contract-sdk/game_base.hpp (hunk 2 above)
Comment (alrxy) [reply sdk/include/game-contract-sdk/game_base.hpp]: hmm, it's enough

**Review by justefg — APPROVED** [review]

**Pull request merged**